src/euphorie/deployment/upgrade/alembic/versions/25_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 25
+Revises: 24
+Create Date: 2019-12-04 11:58:00.148524
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '25'
+down_revision = '24'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('account', sa.Column('created', sa.DateTime(), nullable=True))
+    op.execute("UPDATE account set account_type='full' WHERE account_type is Null")
+    op.add_column('group', sa.Column('brand', sa.String(length=64), nullable=True))
```

Comment:
Uhm, this line smells:
```
op.add_column('group', sa.Column('brand', sa.String(length=64), nullable=True))
```
I think you should remove it

Reply:
Hm, right. we don't need brand in Euphorie.

Reply:
More context: it should be already in the DB https://github.com/euphorie/Euphorie/blob/331630d13d0a2577e3f6e0351540248ba75650ba/src/euphorie/deployment/commands/upgradedb.py#L90